source/_components/hive.markdown

```diff
@@ -9,28 +9,33 @@ ha_category:
   - Light
   - Sensor
   - Switch
+  - Water Heater
 ha_release: 0.59
 ha_iot_class: Cloud Polling
 redirect_from:
   - /components/binary_sensor.hive/
   - /components/climate.hive/
+  - /components/water_heater.hive/
```

Comment:
There is no need to create a redirect for this, since the page has not existed before.
Please remove this line.